blocks/table-v2/src/components/grid/grid.tsx

```diff
@@ -0,0 +1,22 @@
+import { DataEditor, DataEditorProps } from "@glideapps/glide-data-grid";
+import "@glideapps/glide-data-grid/dist/index.css";
+import { useRenderGridPortal } from "./use-render-grid-portal";
+
+type GridProps = DataEditorProps;
+
+export const Grid = (props: GridProps) => {
+  useRenderGridPortal();
+
```

Comment:
Wasn't sure where to drop this comment... below a certain number of rows, I believe it is still possible to get left with a weird partial row between the final actual row of cells, and the "new row" row. We should make sure this is flush always.

<img width="760" alt="Capture 2023-03-09 at 17 51 50@2x" src="https://user-images.githubusercontent.com/6226576/224114092-3b463abd-b687-4863-806d-853ddc46b5b6.png">

Reply:
Addressed this issue here https://github.com/hashintel/hash/pull/1946/commits/13b8a7de657b465ab8f27366cc7887841936dfb6